readme.md

```diff
@@ -13,20 +13,22 @@ $ npm install --save get-gravatar
 ## Usage
 
 ```js
-var fs = require('fs');
-var getGravatar = require('get-gravatar');
+const fs = require('fs');
+const getGravatar = require('get-gravatar');
 
-getGravatar('sindresorhus@gmail.com', {size: 200}, function (err, image) {
-	fs.writeFile('sindre.png', image, function (err) {
-		console.log('Successfully downloaded the Gravarar image');
-	});
+getGravatar('sindresorhus@gmail.com', {size: 200}).then(image => {
+	fs.writeFileSync('sindre.png', image);
+
+	console.log('Successfully downloaded the Gravarar image');
```

Comment:
Not sure about the example. Used `fs.writeFileSync` to avoid callbacks and didn't want to use pify because it would make it more complex to understand.